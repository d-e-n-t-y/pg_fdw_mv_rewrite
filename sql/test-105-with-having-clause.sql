explain (VERBOSE, COSTS OFF) select key, COUNT (value) from test group by key having count (value) > 100;

select key, COUNT (value) from test group by key having count (value) > 100;
