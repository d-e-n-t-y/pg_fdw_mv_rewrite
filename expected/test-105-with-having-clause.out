explain (VERBOSE, COSTS OFF) select key, COUNT (value) from test group by key having count (value) > 100;
                        QUERY PLAN                        
----------------------------------------------------------
 Custom Scan (MVRewriteScan)
   Output: key, (count(value))
   Rewritten: scan of public.test_mv1 WHERE (count > 100)
   ->  Seq Scan on public.test_mv1
         Output: key, count
         Filter: (test_mv1.count > 100)
(6 rows)

select key, COUNT (value) from test group by key having count (value) > 100;
 key  |  count  
------+---------
 key2 | 1000000
(1 row)

